packages/api/src/models/taskModel.js

```diff
@@ -232,6 +232,25 @@ class TaskModel extends BaseModel {
       `,
     );
   }
+
+  /**
+   * Gets the tasks that are due today for a given user
+   * @param {string} userId user id
+   * @returns {Object}
+   * @static
+   * @async
+   */
+  static async getTasksDueTodayFromUserId(userId) {
```

Comment:
As mentioned, this should be for user and farm. I think that this function could return a boolean: are there any tasks due today for this user on this farm? The caller only looks at the length of the returned array.